.storybook/stories/checkbox-toggle/checkbox-toggle-container.stories.ts

```diff
@@ -59,19 +43,94 @@ const defaultParameters: Parameters = {
   },
 };
 
-setupStorybook(ClrCheckboxModule, defaultStory, defaultParameters, generateVariants());
+const CheckboxToggleContainerTemplate: Story = args => {
+  const containerSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-container' : 'clr-toggle-container';
+  const wrapperSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-wrapper' : 'clr-toggle-wrapper';
+  const directive = args.type === CheckboxType.Checkbox ? 'clrCheckbox' : 'clrToggle';
 
-function generateVariants() {
-  const variants: Parameters[] = [];
+  return {
+    template: `
+      <${containerSelector} [clrInline]="clrInline">
+        <label>{{label}}</label>
+        <${wrapperSelector} *ngFor="let _ of createArray(optionCount); let i = index">
+          <input type="checkbox" ${directive} />
+          <label>Option {{i + 1}}</label>
+        </${wrapperSelector}>
+      </${containerSelector}>
+    `,
+    props: args,
+  };
+};
 
-  for (const type of [CheckboxType.Checkbox, CheckboxType.Toggle]) {
-    for (const clrInline of [false, true]) {
-      variants.push({
-        clrInline,
-        type,
-      });
-    }
-  }
+const CheckBoxAllTemplate: Story = args => {
+  const containerSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-container' : 'clr-toggle-container';
+  const wrapperSelector = args.type === CheckboxType.Checkbox ? 'clr-checkbox-wrapper' : 'clr-toggle-wrapper';
+  const directive = args.type === CheckboxType.Checkbox ? 'clrCheckbox' : 'clrToggle';
```

Comment:
This code is repeated. I would extract to a function that can be called like this:

```ts
const { containerSelector, wrapperSelector, directive } = getSelectors(args.type);
```